metrics/expvar/expvar.go

```diff
@@ -75,9 +81,35 @@ type histogram struct {
 	hist *hdrhistogram.WindowedHistogram
 
 	name   string
+	count  metrics.Counter
 	gauges map[int]metrics.Gauge
 }
 
+// Name returns the name of the histogram
+func (h *histogram) Name() string {
+	return h.name
+}
```

Comment:
Yes, it would change the nature of the metrics interfaces. Currently they are producer-only interfaces, and the consumption is assumed to occur behind-the-scenes. Let me think about this a bit.